src/Emojis/emoji.php

```diff
@@ -30,5 +30,38 @@
     "kissing_smiling_eyes"      => "\u{1F619}",
     "kissing_closed_eyes"       => "\u{1F61A}",
     "relaxed"                   => "\u{263A}",
-
-];
\ No newline at end of file
+    "relieved"                  => "\u{1F60C}",
+    "stuck_out_tongue"          => "\u{1F61C}",
+    "stuck_out_tongue_closed_eyes" => "\u{1F61D}",
+    "disappointed"              => "\u{1F61E}",
+    "angry"                     => "\u{1F620}",
```

Comment:
Please can you ensure they are ordered appropriately via the unicode value? You actually ordered a greater part of it but I think "relaxed" should be the last, "relieved" should come before "kissing"